tests/dbaas_policy.robot

```diff
@@ -7,27 +7,201 @@ Resource            ../resources/keywords/dbaas_policy.resource
     Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Project Admin To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-04
+    Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
+    When User Navigates To Import MongoDB Provider Account On Developer View
+    Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Service Admin To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-05
+    Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
+    When User Navigates To Import MongoDB Provider Account On Developer View
+    Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Developer To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-06
+    Given The User Logs In To Openshift As OCP_LDAP_DEV
+    When User Navigates To Add Topology Screen From Developer View      MongoDB     ${DBaaSTestNS}     True
+    Then Cloud Hosted Database Tile Not Loaded Under Add Topology Screen
+
+Scenario: Create DBaaSPolicy As Project Admin From Error screen on Admin View And Imports MongoDB
+    [Tags]  dbaas-test-07
+    Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_PROJECT_ADM
+    Then User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+Scenario: Create DBaaSPolicy As Service Admin From Error screen on Admin View And Imports MongoDB
+    [Tags]  dbaas-test-08
+    Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_SERVICE_PROJECT_ADM
+    Then User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+Scenario: Create DBaaSPolicy As Developer From Error screen on Admin View
+    [Tags]  dbaas-test-09
+    Given The User Logs In To Openshift As OCP_LDAP_DEV
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_DEV
+    Then DBaaSPolicy Creation Forbidden
+
+Scenario: Create DBaaSPolicy As Project Admin From Error screen on Admin View And Imports CrunchyDB
+    [Tags]  dbaas-test-10
+    Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_LDAP_PROJECT_ADM
+    Then User Is Able To Import CrunchyDB Provider Account For ${DBaaSTestNS}
+
+Scenario: Create DBaaSPolicy As Service Admin From Error screen on Admin View And Imports CrunchyDB
+    [Tags]  dbaas-test-11
+    Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
+    When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_SERVICE_PROJECT_ADM
+    Then User Is Able To Import CrunchyDB Provider Account For ${DBaaSTestNS}
+
```

Comment:
I think the developer test case is missing under Crunchy...

Reply:
"Scenario: Create DBaaSPolicy As Developer From Error screen on Admin View" is not ISV dependent and in test script with tag - dbaas-test-09 we are verifying whether the error is displayed and not importing any provider accounts.

Reply:
OK got it.